.github/workflows/python-app.yml

```
@@ -13,24 +13,64 @@
     - name: Run integration tests
       working-directory: ./backend
       run: make test-integration
+    
+    - name: Run tests with coverage
+      working-directory: ./backend
+      run: |
+        TESTING=true . .venv/bin/activate && python -m pytest tests/ --cov=tarsy --cov-report=html --cov-report=xml --cov-report=term-missing
+    
+    - name: Upload coverage reports to Codecov
+      if: github.event_name == 'pull_request' || github.ref == 'refs/heads/master'
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)